gateway-protocol-impl/README.md

```diff
@@ -8,11 +8,11 @@ Zeebe client grpc protobuf protocol.
 
 To generate the go grpc stubs you need the following tools:
 
-1. Go >= 1.7 (https://golang.org/dl/)
+1. Go >= 1.19 (https://golang.org/dl/)
 2. The go protoc generator
 
    ```
-   go get -u github.com/golang/protobuf/protoc-gen-go
+   go install github.com/golang/protobuf/protoc-gen-go@v1.5.3
```

Comment:
🔧 In general makes sense to have this version set here, but not sure whether this doesn't get out dated fast.

Reply:
it's deprecated and no developed further :D the install command complained when no version was provided:
```
> go install github.com/golang/protobuf/protoc-gen-go
go: 'go install' requires a version when current directory is not in a module
```

Reply:
I guess latest would also work 🤔

Reply:
ah, TIL, adjusted :)